### PR TITLE
PID: autoenabling the tables needed for analysis

### PR DESCRIPTION
**Pull request by njacazio**
@jgrosseo this should be the way to go, of course I kept the Configurable flags for debugging purposes to override the workflow and for performance testing.
Thanks to @ktf for the help!

### CODE REVIEW

**Comment by njacazio** [comment]
Hi @vkucera after this is merged we can in principle get rid of the flags for the PID in the validation sw

File: Analysis/Tasks/PID/pidTPC_tiny.cxx [hunk 1]
```diff
@@ -242,7 +266,7 @@ WorkflowSpec defineDataProcessing(ConfigContext const& cfgc)
 {
   auto workflow = WorkflowSpec{adaptAnalysisTask<pidTPCTaskTiny>(cfgc, TaskName{"pidTPC-tiny-task"})};
   if (cfgc.options().get<int>("add-qa")) {
-    workflow.push_back(adaptAnalysisTask<pidTPCTaskQA>(cfgc, TaskName{"pidTOFQA-task"}));
+    workflow.push_back(adaptAnalysisTask<pidTPCTaskQA>(cfgc, TaskName{"pidTPCQA-task"}));
```
Comment (jgrosseo) [inline Analysis/Tasks/PID/pidTPC_tiny.cxx]: can we drop TaskName?

File: Analysis/Tasks/PID/pidTPC.cxx [hunk 2]
```diff
@@ -236,7 +260,7 @@ WorkflowSpec defineDataProcessing(ConfigContext const& cfgc)
 {
   auto workflow = WorkflowSpec{adaptAnalysisTask<pidTPCTaskSplit>(cfgc, TaskName{"pidTPC-split-task"})};
   if (cfgc.options().get<int>("add-qa")) {
-    workflow.push_back(adaptAnalysisTask<pidTPCTaskQA>(cfgc, TaskName{"pidTOFQA-task"}));
+    workflow.push_back(adaptAnalysisTask<pidTPCTaskQA>(cfgc, TaskName{"pidTPCQA-task"}));
   }
```
Comment (jgrosseo) [inline Analysis/Tasks/PID/pidTPC.cxx:264]: can we drop TaskName

**Comment by jgrosseo** [comment]
Looks good. I have two minor comments, and please fix the checkcode errors from the fullCI

File: Analysis/Tasks/PID/pidTPC_tiny.cxx (hunk 1 above)
Comment (njacazio) [reply Analysis/Tasks/PID/pidTPC_tiny.cxx]: Well yes we do but having `TOF` in the task name means that it gets parsed into `t-o-f` that's the only reason that I would like to keep it

**Comment by njacazio** [comment]
This should take care of the fullCI https://github.com/AliceO2Group/AliceO2/pull/6005

File: Analysis/Tasks/PID/pidTPC_tiny.cxx (hunk 1 above)
Comment (jgrosseo) [reply Analysis/Tasks/PID/pidTPC_tiny.cxx]: OK. Then let's keep it

**Comment by jgrosseo** [comment]
codechecker fixed in https://github.com/AliceO2Group/AliceO2/pull/6005
Merging

**Pull request merged**